portfolio/src/main/webapp/script.js

```diff
@@ -36,10 +36,32 @@ function addRandomNJPerson() {
   const personDescripContainer = document.getElementById('NJ-person-descrip');
   personNameContainer.innerText = person;
   personDescripContainer.innerText = descrip;
-  
 }
 
-function closeRandomizer(){
-  const personContainer = document.getElementById("NJ-person-container");
-  personContainer.style.display = "none"
+/**
+ * Closes the randomizer or the popup
+ */
+function closeWindow(window){
+  if (window == 'random'){
```

Comment:
Use === instead according to https://google.github.io/styleguide/jsguide.html#features-equality-checks.

Reply:
Just an alternative option, you could also use a switch statement: https://www.w3schools.com/js/js_switch.asp